addons/l10n_nl/migrations/3.2/post-migrate_update_taxes.py

```diff
@@ -0,0 +1,32 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    nl_country_id = env['res.country'].search([('code', '=', 'NL')], limit=1).id
+    tags_3b = env['account.account.tag']._get_tax_tags('3b (omzet)', nl_country_id)
+    if not tags_3b:
+        return
+    tags_3bl = env['account.account.tag']._get_tax_tags('3bl (omzet)', nl_country_id)
+    tag_3b_plus = tags_3b.filtered(lambda tag: not tag.tax_negate)
+    tag_3bl_plus = tags_3bl.filtered(lambda tag: not tag.tax_negate)
+    tag_3bl_minus = tags_3bl.filtered(lambda tag: tag.tax_negate)
+    params = [
+        tag_3b_plus.id or -1,
```

Comment:
Why this "or" ? Won't the tag always be present in this version ?

Reply:
From my tests, if the tag has not been used, it will not show up by the ORM

Reply:
Right ! If the tags was used,it gets archived, else we remove it. Forget what I said 0:)